web/modules/custom/mymodule/mymodule.module

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+*@file
+*This file implements how to implement the hoook function.
+*/
+
+/**
+*Implements hook_cron
+*/
+function mymodule_cron() {
```

Comment:
Is this working for you?

Reply:
Yes.
![Screenshot from 2023-06-24 18-27-53](https://github.com/Arijit-Acharya-innoraft/customModule/assets/122612545/519d15f6-99d7-42a5-8d94-143aeece474d)